implementations/rust/ockam/ockam_command/src/tcp/util.rs

```diff
@@ -8,3 +20,27 @@ pub fn alias_parser(arg: &str) -> Result<String> {
         Ok(arg.to_string())
     }
 }
+
+pub async fn fetch_list<T: TerminalWriter, L: for<'b> minicbor::Decode<'b, ()>>(
```

Comment:
I meant with this to use it in other commands to fetch the needed recourses list instead of duplicated code.
I wanna use it in the `inlet list` command and other list and delete commands. Do you think that there's a better way?

Reply:
You have an example here on how this was solved for the `node` commands: https://github.com/build-trust/ockam/pull/6665